### PR TITLE
DoE LEAD References Need to be Updated

### PR DESCRIPTION
**Pull request by ahlusar1989**
The below comments and changes are to address the following [bug](https://github.com/usds/justice40-tool/issues/975) that prevents from the data processing pipeline to complete.

I have made inline comments to temporarily identify where changes need to be made.

### CODE REVIEW

File: data/data-pipeline/data_pipeline/etl/score/constants.py [hunk 1]
```diff
@@ -82,7 +82,12 @@
     "Life expectancy (years) (percentile)",
     "Traffic proximity and volume (percentile)",
     "FEMA Risk Index Expected Annual Loss Score (percentile)",
-    "Energy burden (percentile)",
+    # this column is not coherent and is rather vague and should be changed to the original
+    # The question is where do we change this? I have thoughts on this
+    # but am seeking definitive feedback that is aligned with how this
+    # codebase was originally created
+    "BURDEN",
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/score/constants.py:89]: @ahlusar1989 -- you should just rename this in the ETL for DOE LEAD. I should've noticed this on the original PR, but you'll need to update these lines: 

```

        output_df = self.raw_df.rename(
            columns={
                "AvgEnergyBurden": self.ENERGY_BURDEN_FIELD_NAME,
                self.TRACT_INPUT_COLUMN_NAME: self.GEOID_TRACT_FIELD_NAME,
            }
        )
```

Comment (lucasmbrown-usds) [reply data/data-pipeline/data_pipeline/etl/score/constants.py:89]: BTW I just created this for the backlog, after Tuesday we can do some cleanup: https://github.com/usds/justice40-tool/issues/977.

File: data/data-pipeline/data_pipeline/score/field_names.py [hunk 2]
```diff
@@ -87,7 +87,7 @@
 )
 
 # Energy
-ENERGY_BURDEN_FIELD = "Energy burden"
+ENERGY_BURDEN_FIELD = "BURDEN"
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/score/field_names.py:90]: This can go back to `Energy burden` once you finish the renaming.

**Review by lucasmbrown-usds — CHANGES_REQUESTED** [review]
Some good directions -- I left some feedback answering your questions!

File: data/data-pipeline/data_pipeline/etl/score/constants.py (hunk 1 above)
Comment (ahlusar1989) [reply data/data-pipeline/data_pipeline/etl/score/constants.py:89]: Cool - thanks for confirming @lucasmbrown-usds. Just wanted to make sure this was what needed to update here.

File: data/data-pipeline/data_pipeline/score/field_names.py (hunk 2 above)
Comment (ahlusar1989) [reply data/data-pipeline/data_pipeline/score/field_names.py:90]: Yup

File: data/data-pipeline/data_pipeline/etl/score/constants.py (hunk 1 above)
Comment (lucasmbrown-usds) [reply data/data-pipeline/data_pipeline/etl/score/constants.py]: @ahlusar1989 -- why did you mark this as resolved? For code clarity we still need to rename `"BURDEN"` to self.ENERGY_BURDEN_FIELD_NAME.

File: data/data-pipeline/data_pipeline/etl/sources/doe_energy_burden/etl.py [hunk 3]
```diff
@@ -56,6 +56,7 @@ def transform(self) -> None:
 
         output_df = self.raw_df.rename(
             columns={
+                # Where is this referenced elsewhere? It's not immediately clear
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/sources/doe_energy_burden/etl.py]: I'm not sure what you mean by this. The field gets renamed to `self.ENERGY_BURDEN_FIELD_NAME`, and `self.ENERGY_BURDEN_FIELD_NAME` is used frequently elsewhere.

**Review by lucasmbrown-usds — CHANGES_REQUESTED** [review]
Looking better! There's one comment that's the same as before; we want to rename the BURDEN field.

File: data/data-pipeline/data_pipeline/etl/sources/doe_energy_burden/etl.py [hunk 4]
```diff
@@ -56,7 +56,7 @@ def transform(self) -> None:
 
         output_df = self.raw_df.rename(
             columns={
-                "AvgEnergyBurden": self.ENERGY_BURDEN_FIELD_NAME,
+                "BURDEN": self.ENERGY_BURDEN_FIELD_NAME,
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/sources/doe_energy_burden/etl.py:59]: 🎉

**Review by lucasmbrown-usds — APPROVED** [review]
LGTM!

**Pull request merged**